service/src/main/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperService.java

```diff
@@ -193,6 +202,85 @@ public void addDirectionsToCaseDetails(CaseDetails caseDetails,
         });
     }
 
+    @SuppressWarnings("unchecked")
+    public List<Element<Direction>> getDirectionsForUnrepresentedParties(CaseDetails caseDetails,
+                                                                         List<Element<Direction>> directions) {
+        final boolean direction = true;
+
+        return directions.stream()
+            .filter(element -> {
+                    List<Element<UUID>> ids = new ArrayList<>();
+
+                    List<Element<Representative>> representatives = defaultIfNull(
+                        (List<Element<Representative>>) caseDetails.getData().get("representatives"), emptyList());
+
+                    // if assigned to specific person and representatives exist -> find representative ids
+                    // if not -> return direction.
+                    if (directionIsValidForFiltering(element, representatives)) {
+                        if (element.getValue().getParentsAndRespondentsAssignee() != null) {
+                            int assigneeValue = element.getValue().getParentsAndRespondentsAssignee().ordinal();
+
+                            List<Element<Respondent>> respondents =
+                                (List<Element<Respondent>>) caseDetails.getData().get("respondents1");
+
+                            // if respondent exist -> set ids.
+                            // else return direction
+                            if (assigneeValue < respondents.size()) {
+                                ids = respondents.get(assigneeValue).getValue().getRepresentedBy();
+                            } else {
+                                return direction;
+                            }
+                        }
+
+                        if (element.getValue().getOtherPartiesAssignee() != null) {
```

Comment:
Wouldn't else if be better in that scenario? 🤔 we can't have both other and respondent in single direction, can we?